src/commands/pack/deb.ts

```diff
@@ -51,14 +51,15 @@ export default class PackDeb extends Command {
 
   static flags = {
     root: Flags.string({char: 'r', description: 'path to oclif CLI root', default: '.', required: true}),
+    input: Flags.string({ char: 'i', description: 'optionally specify a path to a tarball already generated by NPM', required: false })
```

Comment:
Could you rename this flag to something more descriptive? perhaps `tarball`?

Reply:
@mdonnalley done, thank you for review!